cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/AddCommand.java

```diff
@@ -129,7 +129,7 @@ public void execute() {
         if (!matches) {
             CommandUtil.printError(errStream,
                     "Invalid module name : '" + moduleName + "' :\n" +
-                            "Module name can only contain alphanumerics, underscores and periods " +
+                            "Module name can only contain alphanumerics, underscores " +
```

Comment:
modules can contain a period

Reply:
Changed to use `validateModuleName` method and fixed the error message